services/src/pro/layers/postgres_layer_db.rs

```diff
@@ -58,6 +58,64 @@ where
         Ok(layer_id)
     }
 
+    async fn update_layer(&self, id: &LayerId, layer: UpdateLayer) -> Result<()> {
```

Comment:
add tests for the fns?

Reply:
they are tested through the API handlers.

Reply:
they are tested through the API handlers.